static/styles/zulip.scss

```diff
@@ -1514,9 +1513,10 @@ div.focused_table {
             text-overflow: ellipsis;
             white-space: nowrap;
             .emoji {
-                width: 15px;
-                height: 15px;
-                padding-bottom: 1px;
+                margin: 0;
+                padding: 0;
+                width: 16px;
+                height: 16px;
```

Comment:
If we use the `rendered_markdown` CSS class, the need for this custom rule not be much, since the `.emoji` rendering in message bodies has shrunk.